relays/messages-relay/Cargo.toml

```diff
@@ -11,10 +11,13 @@ async-trait = "0.1.40"
 futures = "0.3.5"
 hex = "0.4"
 log = "0.4.11"
-num-traits = "0.2"
 parking_lot = "0.11.0"
 
 # Bridge Dependencies
 
 bp-message-lane = { path = "../../primitives/message-lane" }
 relay-utils = { path = "../utils" }
+
+# Substrate Dependencies
+
+frame-support = "2.0"
```

Comment:
Seems that we only need the `Weight` type, right? Could we just use our own typedef for this? I'd rather avoid depending on substrate too much.

Reply:
Might be risky if Substrate ever changes their definition of `Weight`, no?

Reply:
Yeah, but this is something that wouldn't be done unnoticed.